src/renderer/components/kube-object/kube-object-details.tsx

```diff
@@ -99,11 +100,7 @@ export class KubeObjectDetails extends React.Component {
   }
 
   @computed get object() {
-    const store = apiManager.getStore(this.path);
-
-    if (store) {
-      return store.getByPath(this.path);
-    }
+    return apiManager.getStore(this.path)?.getByPath(this.path);
```

Comment:
Idk how changes within this file are related to PR. Looks better but are these added here by accident?

Reply:
Yes by accident. I'll revert into a separate PR.